traditions.json

```diff
@@ -0,0 +1,15 @@
+traditions = {
```

Comment:
This isn't a valid JSON document. You can remove `traditions = ` to fix this.

Take a look at some of our other [resource JSON files](https://github.com/python-discord/seasonalbot/tree/master/bot/resources) for some more examples.

Reply:
Fixed, was a copy and paste problem.